rage/Cargo.toml

```diff
@@ -66,11 +66,11 @@ pinentry = "0.5"
 rust-embed = "6"
 
 # rage-mount dependencies
-fuse_mt = { version = "0.5.1", optional = true }
+fuse_mt = { version = "0.6.0", optional = true }
 libc = { version = "0.2", optional = true }
 tar = { version = "0.4", optional = true }
-time = { version = "0.1", optional = true }
-zip = { version = "0.5.9", optional = true }
+time = { version = "0.3.7", optional = true }
```

Comment:
I initially ran into an issue here because `time` is bumping their MSRV in point releases. However that was encountered because the `Cargo.lock` file hadn't been updated. I've pushed a commit to do this.

Reply:
Hmm, this conflicts with `main`. I'll rebase to fix.

Reply:
I initially didn't include `Cargo.lock` into this PR since (as you said) it would introduce conflicts while the PR was being reviewed.